_posts/en/2023-10-22-kafka-1.5.md

```diff
@@ -0,0 +1,50 @@
+---
+layout:    post
+title:     "Kouncil is now more secure"
+date:      2023-10-22 6:00:00 +0100
+published: true
+didyouknow: false
+lang: en
+lang-ref:  kouncil-1.5
+interpreter: Justyna Wiatrzyk-Guzik
+author:    pbelke
+image:     /assets/img/posts/2023-10-22-kouncil-1.5/kouncil-1.5.png
+description: "The latest version of Kouncil focuses on improving application security. Additionally, it introduces the option of securely connecting to Kafka brokers using the SASL authentication mechanism."
+tags:
+- kouncil
+- programming
+- kafka
+- authentication
+- brokers
+---
+
+Working on the latest version of Kouncil, we focused on improving the application security and added the possibility of safely connecting with Kafka brokers.
+
+We discuss both issues in more detail below.
+
+## Autentykacja
+When you open the latest version of Kouncil, you will see a completely new start screen with a login function.
+We have prepared 4 authentication options:
+* In-Memory (with a predefined administrative user)
+* Active Directory
+* LDAP
+* SSO (based on GitHub)
+
+![2019-09-05-Kafka-DLQ-Strategy01.png](/assets/img/posts/2023-10-22-kouncil-1.5/kouncil-1.5-1.png)
```

Comment:
To co się znajduje w nawiasach kwadratowych to tekst alternatywny (wyświetlany kiedy nie można wyświetlić obrazka), w założeniu powinien opisywać to co jest na obrazku, niestety jekyll nie wspiera dodatkowych etykiet pod obrazkami, więc jeśli chcesz taką mieć to należy dodać ją w elemencie z klasą .img-legend (dzięki temu mamy pewność, że wszystkie etykiety we wszystkich postach będą wygladać tak samo), czyli w twoim przypadku powinno to wyglądać tak:
```
![Login screen for SSO authentication](/assets/img/posts/2023-10-22-kouncil-1.5/kouncil-1.5-2.png)
<span class="img-legend">Login screen for SSO authentication</span>
```